examples/quickstart/client/client.csproj

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
-    <ImplicitUsings>disable</ImplicitUsings>
+    <TargetFramework>net8.0</TargetFramework>

```

Comment:
Why are these necessary? It should work without those changes (at least it does locally).

Reply:
I've seen some of our libs complain about net framework mismatch, no idea if they apply here, happy to revert.

Reply:
Please do, and then we'll land this PR since all the other changes are relevant and needed.